android/app/build.gradle

```diff
@@ -142,6 +142,9 @@ android {
 }
 
 dependencies {
+    compile "com.google.android.gms:play-services-base:9.8.0"
+    compile "com.google.android.gms:play-services-maps:9.8.0"
```

Comment:
Do you need these, aren't they already a sub-dependency of of `react-native-maps`?